Overlays/Windows/Libraries/hashing/CMakeLists.txt

```diff
@@ -0,0 +1,6 @@
+append_to_overlay_var(WN_HASHING_XXHASH_COMPILE_OPTIONS /wd4255)
+append_to_overlay_var(WN_HASHING_XXHASH_COMPILE_OPTIONS /wd4668)
+append_to_overlay_var(WN_HASHING_XXHASH_COMPILE_OPTIONS /wd4711)
+append_to_overlay_var(WN_HASHING_XXHASH_COMPILE_OPTIONS /wd4820)
+append_to_overlay_var(WN_HASHING_XXHASH_COMPILE_OPTIONS /wd5045)
+append_to_overlay_var(WN_HASHING_XXHASH_COMPILE_OPTIONS /Qspectre)
```

Comment:
Do we really want to turn on spectre mitigation?
Might be slower than we really want/need.

Reply:
Yeah I wasn't sure. I can remove them. I don't think we care. We can always turn it in later. I'll remove it

Reply:
Sounds good.

Reply:
Removed